CDC-Data-Reconciliation-Frontend/src/components/Report.jsx

```diff
@@ -617,7 +619,7 @@ export default function Report({ reportID }) {
               />
               <button
                 type="button"
-                className="bg-blue-400 text-white px-5 py-2 rounded-md hover:bg-blue-600 flex flex-row items-center justify-around gap-2"
+                className="bg-[#7aa2c4] text-white px-5 py-2 rounded-md hover:bg-[#4c80ae] flex flex-row items-center justify-around gap-2"
```

Comment:
I like this addition, the new color change is warm and improves the feel of the web app.